phpstan.neon

```diff
@@ -28,6 +28,8 @@ parameters:
         - '#Method Rector\\BetterReflection\\Reflector\\SmartClassReflector::reflect\(\) should return Rector\\BetterReflection\\Reflection\\ReflectionClass\|null but returns Rector\\BetterReflection\\Reflection\\Reflection#' # 1
         - '#Parameter \#1 \$node of method Rector\\NodeTypeResolver\\NodeTypeResolver::resolve\(\) expects PhpParser\\Node, PhpParser\\Node\\Expr|string given#' # 1
         - '#Cannot call method render\(\) on phpDocumentor\\Reflection\\DocBlock\\Tag\|string#' # 1
+        - '#Parameter \#1 \$visitor of method PhpParser\\NodeTraverser::addVisitor\(\) expects PhpParser\\NodeVisitor, PhpParser\\NodeVisitor|Rector\\Contract\\Rector\\RectorInterface given#'
+        - '#Method Rector\\NodeTraverser\\RectorNodeTraverser::getRectors\(\) should return array<PhpParser\\NodeTraverser\|Rector\\Contract\\Rector\\RectorInterface> but returns array<PhpParser\\NodeVisitor>#'
```

Comment:
This PHPStan exceptions are growing fast :cry:

Reply:
Well it's not perfect.

In this particular case we might extends from `NodeVisitor` interface, but IIRC that would meant some coupling and empty methods. Shall verify

Reply:
Done, it works :+1:

Reply:
Yes, I know, just comment that I'll try to reduce it again in the future :smile: